cucoslib/workers/init.py

```diff
@@ -15,6 +15,9 @@ def execute(self, arguments):
         self._strict_assert(arguments.get('version'))
         self._strict_assert(arguments.get('ecosystem'))
 
+        # make sure we store package name based on ecosystem package naming case sensitivity
+        arguments['name'] = case_sensitivity_transform(arguments['name'])
```

Comment:
`case_sensitivity_transform` takes two arguments `(ecosystem, name)` , but here only one argument is provided `arguments['name']`. Is this correct ?